utils/log/logger.go

```diff
@@ -19,18 +19,13 @@ const (
 
 func init() {
 	if Logger == nil {
-		Logger = NewLogger()
+		Logger = NewLogger("")
 	}
 }
 
-func NewLogger() Log {
+func NewLogger(logLevel string) Log {
```

Comment:
It is probably better for this function to receive a log level enum, instead of a string.